apps/igloo-cli-npm/README.md

```diff
@@ -12,16 +15,12 @@ The Igloo CLI is available as an NPM or Pip package. To install the CLI, run the
 npm install -g @igloo-kit/cli
 ```
 
-or 
-
-```bash
-pip install igloo-cli
-```
```

Comment:
Why did you remove python?

Reply:
Because this is the readme only for NPM - we can add to the main readme if you want

Reply:
Added it to the main readme